### PR TITLE
Add solutions to readme

### PR DESCRIPTION
**Pull request by polinamarchenko**

### CODE REVIEW

File: big_o_exercise/readme.md [hunk 1]
```diff
@@ -45,6 +51,9 @@ function logAtLeast10(n) {
     }
 }
 
+time complexity: O(n)
+space complexity: O(n)
```
Comment (mmmaaatttttt) [inline big_o_exercise/readme.md]: Careful - your time complexity looks good here, but double-check your space complexity.

**Comment by mmmaaatttttt** [comment]
Hey @polinamarchenko -- double-check one of your answers in Part 2. Otherwise everything looks good. Let me know if you have any questions!

**Comment by polinamarchenko** [comment]
@mmmaaatttttt hey, Matt, not sure why it is not correct. Since it will be console log minimum of 10 times, but could be also a huge number, so output depends on n in this case.

**Comment by mmmaaatttttt** [comment]
Hey @polinamarchenko - what you're describing absolutely makes sense for the time complexity. But you've got `O(n)` as the space complexity as well. You've got `O(1)` as the space complexity for `logUpTo`, which makes sense to me, so shouldn't `logAtLeast10` be the same?

**Pull request closed without merging**